checklist.md

```diff
@@ -0,0 +1,20 @@
+# Checklists
+
+## Suspending Someone
+
+1. Suspend on GitHub
+2. Suspend on Matrix with mjolnir
+3. Suspend on Discourse
+4. Remove inappropriate content
+5. Add entry to moderation-log.md
+  - Include the length of the suspension, if not indefinite
+  - Do __not__ include why the suspension was issued
+
+### Indefinite Suspensions
```

Comment:
Indefinite suspensions should only happen if we have a quorum in the moderation team.

Reply:
That does not align with the most recent harassment case by a troll.